packages/genesys-spark-components/src/components/beta/gux-date-time/gux-date-time.tsx

```diff
@@ -41,16 +44,34 @@ export class GuxDateTime {
 
   componentWillLoad(): void {
     trackComponent(this.root);
-    this.formatter = new DateTimeFormatter(getDesiredLocale(this.root));
+    if (readRegionalDatesCookie()) {
+      this.hasRegionalDatesCookie = true;
+    } else {
+      this.formatter = new DateTimeFormatter(getDesiredLocale(this.root));
+    }
   }
 
-  render() {
-    return (
-      <Host>
-        {this.formatter.formatDateTime(new Date(this.datetime), this.format, {
+  private renderDateTime(): JSX.Element {
+    if (this.hasRegionalDatesCookie) {
+      return sparkIntl
```

Comment:
The types do not seem working as expected. Is there an issue with how I am importing `sparkIntl` from `genesys-spark`? Also, I'm not getting these lint failures locally. Does anyone know why?

Reply:
Lint errors are confirmed to have been from importing from `genesys-spark`. I duplicated the intl utilities and added them to the `genesys-spark-components` package and the lint errors were resolved.